debian/control

```diff
@@ -133,44 +339,23 @@ Recommends:
     gnome-power-manager,
     gnome-weather,
     gucharmap,
-    ibus-table-emoji,
-    libavcodec58,
-    libreoffice-calc,
-    libreoffice-impress,
-    libreoffice-writer,
-    pop-shop,
-    pop-transition,
-    popsicle-gtk,
     seahorse,
     simple-scan,
     totem,
     yelp,
-# Last to replace mime information
-    file-roller,
-    com.github.donadigo.eddy,
-# Programs
-    popsicle,
 # Plugins
-    brltty,
-    fonts-noto-color-emoji,
     gnome-remote-desktop,
     gnome-shell-extension-prefs,
     gstreamer1.0-vaapi,
     libreoffice-gnome,
-    libreoffice-ogltrans,
-    network-manager-config-connectivity-pop,
-    sessioninstaller,
-# Desktop
-    dbus-broker,
-    hidpi-daemon,
-    flatpak,
-    system76-scheduler,
-    touchegg,
-Conflicts: system76-desktop, pulseaudio, pipewire-media-session
-Description: Pop Desktop Metapackage
+# pop-de virtual package
+Conflicts: pop-de
```

Comment:
Wouldn't _this_ configuration only allow one provider of `pop-de` to be installed at a time, if they all conflict with each other/themselves? The "or group" that was used previously was not exclusive; it simply meant at least one of them needs to be installed to satisfy the dependency. On the other hand, if each provider conflicts with and replaces `pop-de`, that would seem to indicate that only one should be installed at a time.

If you want to use a virtual package instead of an "or group" (which doesn't seem necessary), then shouldn't only the `Provides:` line be necessary?

Reply:
not sure, I haven't tested it yet.

Reply:
still just experimenting myself at the moment